libbeat/common/transport/kerberos/config.go

```diff
@@ -59,19 +64,21 @@ func (t *AuthType) Unpack(value string) error {
 }
 
 func (c *Config) Validate() error {
-	if c.AuthType == AUTH_PASSWORD {
+	switch c.AuthType {
+	case AUTH_PASSWORD:
```

Comment:
nits: Do not use C like screaming identifiers for consts. Plus add the actual type to the const if we already have an enum type. Do we need to export the consts or not?

I understand these have not been introduced in this PR, but would be nice to cleanup existing code a little when we touch it :)

Reply:
TIL the all-caps identifiers are also known as screaming identifiers! 😄